core/templates/services/suggestion-modal.service.spec.ts

```diff
@@ -57,7 +65,7 @@ describe('Suggestion Modal Service', () => {
   it('should cancel suggestion', () => {
     const closeSpy = spyOn(uibModalInstanceMock, 'dismiss').and.callThrough();
     const dismissMessage = 'cancel';
-    sms.cancelSuggestion(uibModalInstanceMock, dismissMessage);
+    sms.cancelSuggestion(uibModalInstanceMock);
```

Comment:
`uibModalInstance` is used in ajs, since the file is migrated should we use ngbModalInstance here ? Ditto in the service as well

Reply:
@srijanreddy98 can let us know whether we should use this or not.

Reply:
@srijanreddy98 PTAL!